### ISSUE TITLE
unicrew ml recommendation

### ISSUE THREAD

**Issue by esthicodes** [khangich/machine-learning-interview#18]
https://www.educative.io/courses/machine-learning-system-design

thanks for the posting

is ml related with the cnn model ?

**Comment by akshaya-hub** [comment]
Yes, it is most widely used in  building image recognition in ML  and mainly the CNN “Convolutional Neural Network” is more commonly listed under deep learning algorithms which is a subset of machine learning and AI
https://www.geeksforgeeks.org/convolutional-neural-network-cnn-in-machine-learning/
you can prefer the above link to know more

**Comment by akshaya-hub** [comment]
Yes, it is most widely used in  building image recognition in ML  and mainly the CNN “Convolutional Neural Network” is more commonly listed under deep learning algorithms which is a subset of machine learning and AI
https://www.geeksforgeeks.org/convolutional-neural-network-cnn-in-machine-learning/
you can prefer the above link to know more
